Events/NearSurfaceEvent.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace EddiEvents
+{
+    public class NearSurfaceEvent : Event
+    {
+        public const string NAME = "Near surface";
+        public const string DESCRIPTION = "Triggered when you enter or depart the gravity well around a surface";
+        public const string SAMPLE = null;
+
+        public static Dictionary<string, string> VARIABLES = new Dictionary<string, string>();
+
+        static NearSurfaceEvent()
+        {
+            VARIABLES.Add("near_surface", "A boolean value. True if you are entering the gravity well and and false if you are leaving");
```

Comment:
Therefore a severe misnomer. Should be something like "isApproachingSurface". Correct naming is *very* important when you are in a high-G situation.

Just because FD made a misnomner doesn't mean we should perpetrate it.